modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCasesAdministrator.testcase

```diff
@@ -107,12 +107,37 @@ definition {
 	}
 
 	@description = "This is a test for LPS-156933 - Verify if as an Administrator I should not be able to add a duplicate case"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test AdminCanAddDuplicateCase {
```

Comment:
Hi @joelrocha68, I think that the test name should be 'AdminCannotAddDuplicatedCase'